recipes/leveldb/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,20 @@
+#include "leveldb/db.h"
+#include <iostream>
+
+int main()
+{
+  leveldb::DB *db;
+  leveldb::Options options;
+  options.create_if_missing = true;
+  leveldb::Status status = leveldb::DB::Open(options, "/tmp/testdb", &db);
```

Comment:
In #1542 I tried to avoid a platform-specific tmp path, but as long as CI is passing, then :man_shrugging: